optimade/server/routers/info.py

```diff
@@ -71,8 +71,9 @@ def get_entry_info(request: Request, entry: str) -> EntryInfoResponse:
     properties = retrieve_queryable_properties(
         schema, queryable_properties, entry_type=entry
     )
-
-    output_fields_by_format = {"json": list(properties.keys())}
+    output_fields_by_format = {}
+    for outputformat in CONFIG.enabled_response_formats:
+        output_fields_by_format[outputformat] = list(properties.keys())
```

Comment:
```suggestion
        output_fields_by_format[outputformat] = list(properties)
```

`.keys()` is unnecessary if you just want a list of all keys (I see we use it above too, could be removed)

Reply:
I have removed the unnecessary .keys() from this file.
It would probably be good to do a regex search for the  "list(*.keys()" pattern, so we can remove these in all our code.